src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -350,6 +354,41 @@ export class StudyViewPageStore {
             )
         );
 
+        this.reactionDisposers.push(
+            reaction(
+                () => this.sharedGroupSet,
+                groups => {
+                    if (Object.keys(groups).length > 0) {
+                        this.showGroupsTooltip = true;
+                    }
+                }
+            )
+        );
+
+        this.reactionDisposers.push(
```

Comment:
again, need explaination.  when user logs in, if there are shared groups, save them, right?

Reply:
yes

Reply:
This reaction is no longer required